modules/ListManager.ts

```diff
@@ -1,5 +1,5 @@
 import db, { List } from './db';
-import { renderLists } from './domUtils';
+import { renderLists, noListMessage } from './domUtils';
 
 class ListManager {
   selectedListId: number | null;
```

Comment:
The `id` is a number, this is true of our other DB items as well. 

It would be better to set this up to be more unique and utilize uuid - in keeping with the spirit of this project, we should use the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API)